dvc/repo/ls.py

```diff
@@ -1,34 +1,63 @@
 import os
+import stat
 
 from dvc.exceptions import PathMissingError, OutputNotFoundError
 
 
 @staticmethod
-def ls(url, target=None, rev=None, recursive=None, outs_only=False):
+def ls(
+    url, target=None, rev=None, recursive=None, outs_only=False,
+):
+    """Methods for getting files and outputs for the repo.
+
+    Args:
+        url (str): The repo url
+        target (str, optional)
+        rev (str, optional)
+        recursive (bool, optional)
+        outs_only (bool, optional): Shows only DVC-artifacts
+
+    Returns:
+        list of `fs_node`
+
+    Notes:
+        `fs_node` is a dictionary with structure
+        {
+            "path_info": PathInfo,
```

Comment:
Maybe let's just return string `path` instead? Users of API shouldn't really care about our PathInfos, esp since PathInfos are not even pathlib's Path strictly speaking :slightly_smiling_face: